src/thunderbots/software/ai/hl/stp/tactic/receiver_tactic.h

```diff
@@ -80,6 +80,14 @@ class ReceiverTactic : public Tactic
     std::unique_ptr<Intent> calculateNextIntent(
         intent_coroutine::push_type& yield) override;
 
+    /**
+     * Finds a feasible shot for the robot, if any
```

Comment:
Define what a "feasible" shot means? How is this different from `calcBestShot...`?